Lesson22/todo-app/aysel/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<title>To-Do App</title>
+		<link rel="stylesheet" type="text/css" href="styles.css" />
+	</head>
+	<body>
+
+
+		<div class="container">
+
+			<h1>To-Do List</h1>
+
+			<form id="toDoForm">
+				<input type="text" placeholder="Enter to do" required id="toDoInput"/>
+				<button type="submit" >submit</button>
```

Comment:
To style elements, it's always good to add a class:
```suggestion
				<button class="btn" type="submit" >submit</button>
```